module-2/app.ts

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+import * as dotenv from 'dotenv';
+import userRouter from './routes/userRouter';
+import { errorHandler, pageNotFound } from './middleware/errorHandlers';
+
+dotenv.config();
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
+  res.redirect('/users');
+});
+
+app.use('/users', userRouter);
+app.use(pageNotFound);
+app.use(errorHandler);
```

Comment:
Please move every router logic to `routes` folder, then export them out from a single file like `index.ts` and use it like this app.use(appRouter)

Reply:
It helps keeping this file clean in the future